PomoTodo/PomoTodo/Sources/Views/Chart/StatisticsView.swift

```diff
@@ -10,7 +10,12 @@ import Charts
 
 // MARK: - 통계 탭 전체 뷰
 struct StatisticsView: View {
-  @StateObject private var viewModel = StatisticsViewModel()
+  @StateObject private var viewModel: StatisticsViewModel
+  
+  init(viewModel: StatisticsViewModel) {
+    _viewModel = StateObject(wrappedValue: viewModel)
+  }
+  
```

Comment:
이거 외부에서 주입받고 싶으시면
`@ObservableObject` 쓰시면 될 것 같습니다.
```suggestion
  @Observableobject private var viewModel: StatisticsViewModel
  
  init(viewModel: StatisticsViewModel) {
     self.viewModel = viewModel
  }
  
```

Reply:
@Monfi98  @ObservableObject 는 이슈 발생해서 ObservedObject로 처리하겠습니다